routes/question-route.js

```diff
@@ -0,0 +1,12 @@
+var express = require ("express");
+var questionRouter  = express.Router();
+var Question = require('../models/question.js');
+
+
+questionRouter.get('/',(req,res)=>{
+    question.findAll().then((questions)=>{
+        res.send(questions)
+    })
```

Comment:
right idea, but need to make sure we use the model variable name to findAll, or else we'd get an error. In this case we would need to use it like this:

```js
questionRouter.get('/',(req,res)=>{
     Question.findAll().then((questions)=>{
         res.send(questions)
     })
})
```